packages/docs/content/patterns/Forms/forms.mdx

```diff
@@ -4,7 +4,7 @@ title: Forms
 
 The design system includes components for the standard HTML form elements: [checkboxes](/components/checkbox/) and [radio buttons](/components/radio/), [text fields](/components/text-field/), [dropdowns ](/components/dropdown/), fieldsets, labels, and so on.
 
-Below are some best practices your project should aim to follow in order to provide the most accessible and usable experience for your users. Additional guidance, specific to each form field, is provided in each component's guidance tab.
+Below are some best practices your project should follow in order to provide the most accessible and usable experience for your users. Additional guidance, specific to each form field, is provided in each component's guidance tab.
```

Comment:
Just curious - what's a "guidance tab"? Is this referring to the guidance sections we have?

Reply:
Oh, it was there originally. Must have been a time when the doc site was structured differently